include/RAJA/policy/PolicyBase.hpp

```diff
@@ -50,6 +50,11 @@ enum class Launch { undefined, sync, async };
 
 enum class Platform { undefined = 0, host = 1, cuda = 2, omp_target = 4 };
 
+enum CudaSync {
```

Comment:
Indeed, I wasn't sure which way to go because as @DavidPoliakoff pointed out, RAJA::sync would be cleaner than RAJA::something::sync. Although, I still think having 2 different "sync"s is mildly confusing, even if they are scoped in different enumeration classes.

The Launch:: enum is an int, so to reuse it, I would need to redefine cuda_reduce's sync/async template parameter to accept an int, which seems unnecessarily complicated.

Would enum class CudaSync{ csync = false; . . .}; be a reasonable compromise?